e2e_test/e2e_test.go

```diff
@@ -36,3 +37,44 @@ func TestSendCelo(t *testing.T) {
 	err = network.AwaitTransactions(ctx, tx)
 	require.NoError(t, err)
 }
+
+func TestSingleNodeManyTxs(t *testing.T) {
+	rounds := 100
```

Comment:
I think rounds has the potential to be confused with rounds in the consensus, I'd go for something like txCount.

Reply:
well this is actually 100 rounds of waits, not really 100txs (even if there are also 100txs, but we could send more than one per wait round)

Reply:
What about iterations,  just not not be confused with consensus rounds?

Reply:
Also I think 100 is too many. When I'm testing repeated behaviour I generally chceck for 3 iterations, the first just to make sure the thing works, the second to know that it can repeat the behaviour and the third just to make sure. So I would go for 3 or 5 if I wanted to be really really sure, but after that I don't think we get much value from the extra rounds.

Reply:
Hi @hbandura, I've made these changes here - https://github.com/celo-org/celo-blockchain/pull/1835/files let me know what you think?

Reply:
merged it!